linker_scripts/soundfont.ld

```diff
@@ -0,0 +1,19 @@
+OUTPUT_ARCH (mips)
+
+/* Soundfont Linker Script, maps data into rodata and adds a file length symbol */
+
+SECTIONS {
+
+    .rodata :
+    {
+        *(.data*)
+        *(.rodata*)
+        . = ALIGN(16);
+        __LEN__ = . - ADDR(.rodata);
```

Comment:
dang, I was hoping we could use the same data_with_rodata linker script for all audio. I changed the audio table one in the z_message/z_game_over PR but maybe that was a mistake. Some options:

* Use the same linker script for everything but include a data length symbol (although I think it should be named something more explicit than __LEN__, like RodataSize or something
* Use the same linker script for everything but use a different method of computing the length (like `objcopy` then `wc`)
* Don't use the same linker script and restore the audio table one

What do you think?

Reply:
I think soundfonts are "weird" enough that they justify their own linker script, while the audio tables and message/gameover have identical needs so sharing a single linker script is fine.